test/battle/ability/synchronize.c

```diff
@@ -0,0 +1,67 @@
+#include "global.h"
+#include "test/battle.h"
+
+SINGLE_BATTLE_TEST("Synchronize will mirror back non volatile status back at opposing mon")
+{
+
+    GIVEN {
+        ASSUME(GetMoveEffect(MOVE_TOXIC) == EFFECT_TOXIC);
+        PLAYER(SPECIES_WOBBUFFET);
+        OPPONENT(SPECIES_ABRA) { Ability(ABILITY_SYNCHRONIZE); }
+    } WHEN {
+        TURN { MOVE(player, MOVE_TOXIC); }
+    } SCENE {
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_TOXIC, player);
+        ANIMATION(ANIM_TYPE_STATUS, B_ANIM_STATUS_PSN, opponent);
+        STATUS_ICON(opponent, badPoison: TRUE);
+        ABILITY_POPUP(opponent, ABILITY_SYNCHRONIZE);
+        ANIMATION(ANIM_TYPE_STATUS, B_ANIM_STATUS_PSN, player);
+        STATUS_ICON(player, badPoison: TRUE);
+    }
+}
+
+SINGLE_BATTLE_TEST("Synchronize will still show up the ability pop up even if it fails")
+{
+    GIVEN {
+        ASSUME(GetMoveEffect(MOVE_TOXIC) == EFFECT_TOXIC);
```

Comment:
```suggestion
        ASSUME(MoveMakesContact(MOVE_TACKLE));
```
Same change for the test below as these are more relevant to these tests